CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 
+* [CHANGE] Reduce permissions on all files and directories created by cortex to just the user it executes as. #3713
```

Comment:
We've just released `1.7.0-rc.0`. Could you rebase `master` and move the CHANGELOG entry under the `master / unreleased` section, please?